Doc/howto/logging.rst

```diff
@@ -774,29 +774,29 @@ What happens if no configuration is provided
 
 If no logging configuration is provided, it is possible to have a situation
 where a logging event needs to be output, but no handlers can be found to
-output the event. The behaviour of the logging package in these
-circumstances is dependent on the Python version.
+output the event.
 
-For versions of Python prior to 3.2, the behaviour is as follows:
+The event is output using a 'handler of last resort', stored in
+:data:`lastResort`. This internal handler is not associated with any
```

Comment:
`lastResort`, `raiseExceptions` etc. are module-level attributes - how come we don't use the ``:attr:`` role for them? Is my understanding of that role outdated?

Reply:
We use `data` to describe module level variables to define: https://www.sphinx-doc.org/en/master/usage/domains/python.html#directive-py-data and reference them: https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-data :)

Reply:
Thanks for updating my knowledge.